tests/lib/newlib/newlib_api/README.txt

```diff
@@ -0,0 +1,88 @@
+Title: Kernel Access to Standard Libraries
+
+Description:
+
+This test verifies access to the Newlib.
```

Comment:
It's better to say like "This test verifies the hook layer of Newlib".

Reply:
Address by your comment.